src/Compilers/CSharp/Portable/Symbols/ErrorTypeSymbol.cs

```diff
@@ -137,6 +137,13 @@ public override IEnumerable<string> MemberNames
         /// returns an empty ImmutableArray. Never returns Null.</returns>
         public override ImmutableArray<Symbol> GetMembers()
         {
+            if (IsTupleType)
+            {
+                var result = AddOrWrapTupleMembers(ImmutableArray<Symbol>.Empty);
```

Comment:
>result [](start = 20, length = 6)

Can this really happen?  Do we have a test that exercises this?

Reply:
Yes, this is used/required by many tests.
Here's the list:

---
In reply to: [356901339](https://github.com/dotnet/roslyn/pull/39370#discussion_r356901339) [](ancestors = 356901339)